spec/operators/toArray-spec.ts

```diff
@@ -1,115 +1,145 @@
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
+/** @prettier */
 import { toArray, mergeMap } from 'rxjs/operators';
 import { of } from 'rxjs';
+import { TestScheduler } from 'rxjs/testing';
+import { observableMatcher } from '../helpers/observableMatcher';
 
 /** @test {toArray} */
-describe('toArray operator', () => {
-  it('should reduce the values of an observable into an array', () => {
-    const e1 =   hot('---a--b--|');
-    const e1subs =   '^        !';
-    const expected = '---------(w|)';
+describe('toArray', () => {
+  let rxTestScheduler: TestScheduler;
 
-    expectObservable(e1.pipe(toArray())).toBe(expected, { w: ['a', 'b'] });
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+  beforeEach(() => {
+    rxTestScheduler = new TestScheduler(observableMatcher);
   });
 
-  it('should be never when source is never', () => {
-    const e1 =  cold('-');
-    const e1subs =   '^';
-    const expected = '-';
+  it('should reduce the values of an observable into an array', () => {
+    rxTestScheduler.run(({ hot, expectObservable, expectSubscriptions }) => {
+      const e1 = hot('  ---a--b--|   ');
+      const e1subs = '  ^--------!   ';
+      const expected = '---------(w|)';
+
+      expectObservable(e1.pipe(toArray())).toBe(expected, { w: ['a', 'b'] });
+      expectSubscriptions(e1.subscriptions).toBe(e1subs);
+    });
+  });
 
-    expectObservable(e1.pipe(toArray())).toBe(expected);
-    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+  it('should be never when source is never', () => {
+    rxTestScheduler.run(({ cold, expectObservable, expectSubscriptions }) => {
+      const e1 = cold(' -');
+      const e1subs = '  ^';
+      const expected = '-';
+
+      expectObservable(e1.pipe(toArray())).toBe(expected);
+      expectSubscriptions(e1.subscriptions).toBe(e1subs);
+    });
   });
 
   it('should be never when source is empty', () => {
```

Comment:
We might was well fix this while we're here:

```suggestion
  it('should be empty when source is empty', () => {
```